dask/array/image.py

```diff
@@ -0,0 +1,55 @@
+from .core import Array
+from glob import glob
+import os
+try:
+    from skimage.io import imread as sk_imread
+except ImportError:
+    pass
+
+
+def tokenize(*args):
+    from hashlib import md5
+    return md5(str(args).encode()).hexdigest()
+
+def add_leading_dimension(x):
+    return x[None, ...]
+
+
+def imread(filename):
+    """ Read a stack of images into a dask array
```

Comment:
@JDWarner I'm happy to yield entirely to your judgement here.  Should we import from `skimage` or `imageio`?  Or should we transition as `skimage` transitions?